roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -194,6 +194,14 @@
 
 # storageclasses are used by default but if static then disable
 # storageclasses with the storageClassName set to "" in pvc.j2
+- name: Check to see if PVC already exists
+  oc_obj:
+    state: list
+    kind: pvc
+    name: "{{ openshift_logging_elasticsearch_pvc_name }}"
+    namespace: "{{ openshift_logging_elasticsearch_namespace }}"
+  register: logging_elasticsearch_pvc
+
```

Comment:
I'll add some comments to the when block to make it more obvious what is being checked for and why